lib/utils/lazy.ts

```diff
@@ -30,3 +31,7 @@ const once = <T>(fn: () => T) => {
 export const getBalenaSdk = once(() =>
 	(require('balena-sdk') as typeof BalenaSdk).fromSharedOptions(),
 );
+
+export const getVisuals = once(
+	() => require('resin-cli-visuals') as typeof visuals,
```

Comment:
NIT: for clarity (code reader's first impression of `lazy.ts`), I would have preferred replacing the module-level import (line 18) with a local import here:
```
	() => require('resin-cli-visuals') as typeof import('resin-cli-visuals'),
```

Reply:
I understand the sentiment but I do believe that `import type` is the right solution for it and I expect it to be usable pretty soon (so it's worth waiting a little bit, but if it was further out I'd definitely agree with the `import()`)